docs/blog/2024-january-update.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: Jan 11, 2023
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - January 2024
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+I'm here with our traditional January community update to give the newsletter team the month off.
+
+2024 is looking to be a refinement of our approach to 2023,
+with a few new experiments thrown in.
+
+Our 2024 conferences
+--------------------
+
+It was lovely to see everyone again in Portland & Australia conferences this year,
```

Comment:
I feel like we could expand more about the experience of returning to in-person conferences, what things seem different and how we adapted to address the current social and professional climate.